src/EnergyPlus/HybridModelConfigFile.cc

```diff
@@ -0,0 +1,276 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <HybridModelConfigFile.hh>
+#include <map>
+#include <math.h>       /* floor */
+#include <string>
+#include <list>
+#include <windows.h>
+#define PERFORMANCE_CURVE_ID_TAG "PERFRM_CRV"
+#define OPERATIONAL_CONTRAINT_ID_TAG "OP_CON"
+#define ENVIRONMENTAL_CONTRAINT_ID_TAG "ENV_CON"
+#define SYSTEM_PERFORMANCE_ID_TAG "SYSTEM_INPUTS"
+//#include <hash>
+//std::fstream infile;
+using namespace std;
+list<string> line;
+vector<double> curve;
+vector<double> OpContraints;
+vector<double> envContraint;
+
+//Rated Msa  (m3/s)	Minium Ventialtion rate Msa  (m3/s)	Rated H (kw)
+
+//list<curve*> PerformanceCurves;
+
+namespace EnergyPlus {
+
+	namespace HybridModelConfigFile {
```

Comment:
This config file must be some sort of temporary thing that would be removed before this goes into E+ right?